Jellyseerr install issue

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Jellyseerr

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/jellyseerr.sh)"

### 📝 Provide a clear and concise description of the issue.

Seems to error out when installing sqlite3 as part of the jellyseer installation

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Run install command and this error appears regardless of advanced or default settings.

### ❌ Paste the full error output (if available).

```console
Downloading @formatjs/swc-plugin-experimental@0.4.0: 8.56 MB/8.56 MB, done
PrDownloading @swc/core-linux-x64-musl@1.6.5: 20.73 MB/20.73 MB, done
Downloading @swc/core-linux-x64-gnu@1.6.5: 16.74 MB/16.74 MB, done
Downloading ace-builds@1.15.2: 9.13 MB/9.13 MB, done
PrDownloading typescript@4.9.5: 11.62 MB/11.62 MB, done
Downloading openpgp@5.11.2: 5.42 MB/5.42 MB, done
PrDownloading next@14.2.24: 20.75 MB/20.75 MB, done
PrDownloading @next/swc-linux-x64-gnu@14.2.24: 41.91 MB/41.91 MB, done
PrDownloading ace-builds@1.35.0: 9.93 MB/9.93 MB, done
Downloading @next/swc-linux-x64-musl@14.2.24: 50.27 MB/50.27 MB, done
PrDownloading @formatjs/intl-displaynames@6.6.8: 6.00 MB/6.00 MB, done
Downloading react-native@0.74.2: 24.04 MB/24.04 MB, done
PrDownloading three@0.165.0: 5.90 MB/5.90 MB, done
PrDownloading @img/sharp-libvips-linuxmusl-x64@1.0.2: 7.20 MB/7.20 MB, done
Downloading @img/sharp-libvips-linux-x64@1.0.2: 7.06 MB/7.06 MB, done
PrDownloading jsc-android@250231.0.0: 32.05 MB/32.05 MB, done
PrProgress: resolved 2151, reused 0, downloaded 2135, added 2151, done
 ⠙node_modules/.pnpm/@swc+core@1.6.5_@swc+helpers@0.5.11/node_modules/@swc/core: Running postinstall script.node_modules/.pnpm/@swc+core@1.6.5_@swc+helpers@0.5.11/node_modules/@swc/core: Running postinstall script,node_modules/.pnpm/cypress@14.1.0/node_modules/cypress: Running postinstall script, done in 401ms
nonode_modules/.pnpm/bcrypt@5.1.0_encoding@0.1.13/node_modules/bcrypt: Running install script, done in 10.7sde_modules/.pnpm/sharp@0.33.4/node_modules/sharp: Running install script, done in 107ms
nonode_modules/.pnpm/sqlite3@5.1.7/node_modules/sqlite3: Running install script, failed in 11.7s
.../sqlite3@5.1.7/node_modules/sqlite3 install$ prebuild-install -r napi || node-gyp rebuild
│ prebuild-install warn install Request timed out
│ gyp info it worked if it ends with ok
│ gyp info using node-gyp@8.4.1
│ gyp info using node@22.14.0 | linux | x64
│ (node:5558) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.ass…
│ (Use `node --trace-deprecation ...` to show where the warning was created)
│ gyp info find Python using Python version 3.11.2 found at "/usr/bin/python3"
│ gyp http GET https://nodejs.org/download/release/v22.14.0/node-v22.14.0-headers.tar.gz
│ gyp http 200 https://nodejs.org/download/release/v22.14.0/node-v22.14.0-headers.tar.gz
│ gyp http GET https://nodejs.org/download/release/v22.14.0/SHASUMS256.txt
│ gyp http 200 https://nodejs.org/download/release/v22.14.0/SHASUMS256.txt
│ gyp info spawn /usr/bin/python3
│ gyp info spawn args [
│ gyp info spawn args   '/opt/jellyseerr/node_modules/.pnpm/node-gyp@8.4.1/node_modules/node-gyp/gyp/gy…
│ gyp info spawn args   'binding.gyp',
│ gyp info spawn args   '-f',
│ gyp info spawn args   'make',
│ gyp info spawn args   '-I',
│ gyp info spawn args   '/opt/jellyseerr/node_modules/.pnpm/sqlite3@5.1.7/node_modules/sqlite3/build/co…
│ gyp info spawn args   '-I',
│ gyp info spawn args   '/opt/jellyseerr/node_modules/.pnpm/node-gyp@8.4.1/node_modules/node-gyp/addon.…
│ gyp info spawn args   '-I',
│ gyp info spawn args   '/root/.cache/node-gyp/22.14.0/include/node/common.gypi',
│ gyp info spawn args   '-Dlibrary=shared_library',
│ gyp info spawn args   '-Dvisibility=default',
│ gyp info spawn args   '-Dnode_root_dir=/root/.cache/node-gyp/22.14.0',
│ gyp info spawn args   '-Dnode_gyp_dir=/opt/jellyseerr/node_modules/.pnpm/node-gyp@8.4.1/node_modules/…
│ gyp info spawn args   '-Dnode_lib_file=/root/.cache/node-gyp/22.14.0/<(target_arch)/node.lib',
│ gyp info spawn args   '-Dmodule_root_dir=/opt/jellyseerr/node_modules/.pnpm/sqlite3@5.1.7/node_module…
│ gyp info spawn args   '-Dnode_engine=v8',
│ gyp info spawn args   '--depth=.',
│ gyp info spawn args   '--no-parallel',
│ gyp info spawn args   '--generator-output',
│ gyp info spawn args   'build',
│ gyp info spawn args   '-Goutput_dir=.'
│ gyp info spawn args ]
│ gyp ERR! build error 
│ gyp ERR! stack Error: not found: make
│ gyp ERR! stack     at getNotFoundError (/opt/jellyseerr/node_modules/.pnpm/which@2.0.2/node_modules/w…
│ gyp ERR! stack     at /opt/jellyseerr/node_modules/.pnpm/which@2.0.2/node_modules/which/which.js:57:18
│ gyp ERR! stack     at new Promise (<anonymous>)
│ gyp ERR! stack     at step (/opt/jellyseerr/node_modules/.pnpm/which@2.0.2/node_modules/which/which.j…
│ gyp ERR! stack     at /opt/jellyseerr/node_modules/.pnpm/which@2.0.2/node_modules/which/which.js:71:22
│ gyp ERR! stack     at new Promise (<anonymous>)
│ gyp ERR! stack     at subStep (/opt/jellyseerr/node_modules/.pnpm/which@2.0.2/node_modules/which/whic…
│ gyp ERR! stack     at /opt/jellyseerr/node_modules/.pnpm/which@2.0.2/node_modules/which/which.js:80:22
│ gyp ERR! stack     at /opt/jellyseerr/node_modules/.pnpm/isexe@2.0.0/node_modules/isexe/index.js:42:5
│ gyp ERR! stack     at /opt/jellyseerr/node_modules/.pnpm/isexe@2.0.0/node_modules/isexe/mode.js:8:5
│ gyp ERR! stack     at FSReqCallback.oncomplete (node:fs:198:21)
│ gyp ERR! System Linux 6.8.12-8-pve
│ gyp ERR! command "/usr/bin/node" "/opt/jellyseerr/node_modules/.pnpm/node-gyp@8.4.1/node_modules/node…
│ gyp ERR! cwd /opt/jellyseerr/node_modules/.pnpm/sqlite3@5.1.7/node_modules/sqlite3
│ gyp ERR! node -v v22.14.0
│ gyp ERR! node-gyp -v v8.4.1
│ gyp ERR! not ok 
└─ Failed in 11.7s at /opt/jellyseerr/node_modules/.pnpm/sqlite3@5.1.7/node_modules/sqlite3
 ELIFECYCLE  Command failed with exit code 1.
 ⠏
[ERROR] in line 46: exit code 0: while executing command $STD pnpm install --frozen-lockfile
```

### 🖼️ Additional context (optional).

_No response_

--- a/install/jellyseerr-install.sh
+++ b/install/jellyseerr-install.sh
@@ -14,11 +14,13 @@ network_check
 update_os
 
 msg_info "Installing Dependencies"
-$STD apt-get install -y curl
-$STD apt-get install -y sudo
-$STD apt-get install -y mc
-$STD apt-get install -y git
-$STD apt-get install -y gpg
+$STD apt-get install -y \
+    curl \
+    gnupg \
+    mc \
+    git \
+    sudo \
+    build-essential
 msg_ok "Installed Dependencies"
 
 msg_info "Setting up Node.js Repository"

